frappe/database/postgres/schema.py

```diff
@@ -40,7 +40,18 @@ def alter(self):
 			query.append("ADD COLUMN `{}` {}".format(col.fieldname, col.get_definition()))
 
 		for col in self.change_type:
-			query.append("ALTER COLUMN `{}` TYPE {}".format(col.fieldname, get_definition(col.fieldtype, precision=col.precision, length=col.length)))
+			using_clause = ""
+			if col.fieldtype in ("Datetime"):
+				# Add USING :: timestamp if type is datetime
+				using_clause = "USING {}::timestamp without time zone".format(col.fieldname)
```

Comment:
Can you give some explanation of why `USING` clause should be used here?
Maybe some use case will explain it better.